src/main/java/com/exercisetracker/exercisetracker/exceptions/CustomExceptionHandler.java

```diff
@@ -0,0 +1,54 @@
+package com.exercisetracker.exercisetracker.exceptions;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ControllerAdvice
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(Exception.class)
```

Comment:
Do you understand what these means abi na copy and paste :thinking:

Reply:
Atleast I have a basic knowledge of how it works and I understand spring AOP from the spring tutorial I did